shaders/paintball.shader

```diff
@@ -6,29 +6,40 @@ uniform float fuzz = 0.0;
 uniform float transparent_color_index = 1.0;
 uniform sampler2D palette: filter_nearest;
 
-@LoadColorShaderComponent
-
-@LoadCircleShaderComponent
 
 varying float v_radius;
-varying float v_total_radius;
+varying float v_total_radius; // @todo unused variable
```

Comment:
There's one unused variable ... maybe delete it ??? But probably later after all is fixed ...